### DISCUSSION TITLE
error: the baseline does not contain an entry for port piper-phonemize

### DISCUSSION THREAD

**Discussion by siddhsql** [microsoft/vcpkg#40849] [Q&A]
I was somehow able to install a custom package that is not published to vcpkg repo into my vcpkg root directory:

```
vcpkg list
fmt:x64-windows                                   11.0.2              {fmt} is an open-source formatting library provi...
piper-phonemize:x64-windows                       bfc2e75499578...    C++ library for converting text to phonemes for ...
spdlog:x64-windows                                1.14.1              Very fast, header-only/compiled, C++ logging lib...
vcpkg-cmake-config:x64-windows                    2024-05-23
vcpkg-cmake:x64-windows                           2024-04-23
```

but when I try to use it in a project like this:

```
{
  "dependencies": [
    "piper-phonemize"
  ]
}
```

I get this error:

```
..\..\vcpkg\vcpkg install
error: the baseline does not contain an entry for port piper-phonemize
```

How can I fix this please?

Also I am trying to understand how vcpkg works. It seems there is a root directory and a `packages` folder in that root that contains the installed packages but then why does `vcpkg` duplicate packages in the project's directory?

when i try to list the packages in my vcpkgroot i get:

```
ls packages
detect_compiler_x64-windows  espeak-ng_x64-windows  fmt_x64-windows  nlohmann-json_x64-windows  spdlog_x64-windows  vcpkg-cmake-config_x64-windows  vcpkg-cmake_x64-windows
```

this list does not contain `piper-phonemize`. what gives? and it contains `espeak-ng_x64-windows` but I don't see it in the output of `vcpkg list`. again what gives here?

**Comment by autoantwort** [comment]
Did you provide an overlay port? https://learn.microsoft.com/en-us/vcpkg/concepts/overlay-ports
